spec/lib/bright_serializer/serializer_spec.rb

```diff
@@ -8,10 +8,17 @@
       Class.new do
         include BrightSerializer::Serializer
         attributes :first_name, :last_name
-        attribute :name do |object|
+        attribute :name do |object, _params|
```

Comment:
Does this change means that when you create a block attributes, you must add `_params` in the argument of the block. I think it will be a big breaking change 🤔

Reply:
I made the changes so it's not required anymore